import_tracker/lazy_import_errors.py

```diff
@@ -248,14 +259,16 @@ def __ge__(self, *_, **__):
     def __get__(self, *_, **__):
         self._raise()
 
-    def __getitem__(self, *_, **__):
+    def __getitem__(self, *args, **kwargs):
         self._raise()
 
     def __gt__(self, *_, **__):
         self._raise()
 
     def __hash__(self, *_, **__):
-        self._raise()
+        if not _is_import_time():
+            self._raise()
+        return randint(0, 100000000)
```

Comment:
I don't like using `randint` here. How about `id(self)`? Nondeterminism scares me.

Reply:
Ah yes. That would be better.. Will change this